packages/auth/src/session.ts

```diff
@@ -180,7 +187,8 @@ export class Session {
         tracker,
         networks,
         contexts,
-        orchestrator
+        orchestrator,
+        projectAccessKey
```

Comment:
Technically there is another place below where this should be added in the new account case, do you mind adding it there as well? Line 279